### ISSUE TITLE
Improve the Encounter Title on the Combat Encounters tab

### ISSUE THREAD

**Issue by mclemente** [foundryvtt/foundryvtt#9515]
### User Experience

TL;DR: Consider renaming "No Active Encounters" to "No Encounters" and change `.combat-sidebar .combat-tracker-header div.encounter-controls h3`'s flex to 2.

Prints are on SWADE system.

The Encounter Title's "No Active Encounter" wraps up to a 3rd line on Portuguese.
![image](https://github.com/foundryvtt/foundryvtt/assets/5288872/11d82dac-abfb-4897-9e8d-a516cffdf833)
Even if we were to use another a smaller word, it would still wrap to the 2nd line like in English. 

Here's the "Not Started" translation (same things happens in English).
![image](https://github.com/foundryvtt/foundryvtt/assets/5288872/c4d719dd-df5b-4459-8572-0832cd13aba9)

I think it would look better if the text didn't wrap and looked like how it looks when combat is on ("Round 1").

Here is how it would look like with `flex: 2`:
![image](https://github.com/foundryvtt/foundryvtt/assets/5288872/23631dd1-ae75-4506-9886-39b741db1672)
And here is how "No Active Encounters" would look if it dropped the "active":
![image](https://github.com/foundryvtt/foundryvtt/assets/5288872/9cccb98a-23c1-4426-8868-604791c827ce)

**Comment by aaclayton** [comment]
Agree it should be just "No Encounters" instead of "No Active Encounters" as this would be more precise. We can look at the flexrow spacing while we're at it.
